### DISCUSSION TITLE
lowering / cost analysis of @nnx.jit functions

### DISCUSSION THREAD

**Discussion by lzanini** [google/flax#4093] [Q&A]
hi, is there a way to use jax's cost analysis api for nnx.jitted functions ? typically, a training step with a module, an optimizer and a metric. it is unclear to me how to lower the inner jax.jitted function.

**Comment by cgarciae** [comment]
Converted this into an issue: #4094
